### PR TITLE
Fix runAsUser chown

### PR DESCRIPTION
**Pull request by sam-utila**
Should also recursivly chown binaryExtractLocation.

### CODE REVIEW

**Review by omer-utila — APPROVED** [review]

File: prepare_database.go [hunk 1]
```diff
@@ -55,6 +55,11 @@ func defaultInitDatabase(binaryExtractLocation, runtimePath, pgDataDir, runAsUse
 			return err
 		}
 
+		err = chownR(binaryExtractLocation, runAsUser)
+		if err != nil {
```
Comment (dor-utila) [inline prepare_database.go:59]: nit: can be one-lined

Comment (dor-utila) [reply prepare_database.go:59]: although it won't be visually similar to the rest of the code in this func so whatever

File: runas.go [hunk 2]
```diff
@@ -57,3 +58,19 @@ func chown(file string, runAsUser string) error {
...
+		return err
+	}
+
+	return filepath.Walk(path, func(name string, info os.FileInfo, err error) error {
+		if err == nil {
+			if err = os.Chown(name, int(uid), int(gid)); err != nil {
+				return fmt.Errorf("unable to chown '%s' file with '%s': %w", name, runAsUser, err)
+			}
+		}
+		return err
```
Comment (dor-utila) [inline runas.go:74]: reverse, to reduce indentation:
```suggestion
		if err != nil {
			return err
		}
		if err = os.Chown(name, int(uid), int(gid)); err != nil {
			return fmt.Errorf("unable to chown '%s' file with '%s': %w", name, runAsUser, err)
		}
		return nil
	
```

**Review by dor-utila — APPROVED** [review]

File: runas.go (hunk 2 above)
Comment (sam-utila) [reply runas.go:74]: ack

**Pull request merged**